torch/csrc/distributed/autograd/context/dist_autograd_container.h

```diff
@@ -18,6 +18,11 @@ namespace autograd {
 // autograd_context_id. The autograd_context_id itself is a 64 bit globally
 // unique id. The first 16 bits is the worker_id and the next 48 bits is an
 // auto-incrementing id for each worker.
+//
+// This container is also responsible for maintaining a globally unique message
+// id, which is used to associate send/recv autograd function pairs. The format
+// is similar to the autograd_context_id where we have a 64 bit integer with
```

Comment:
doesn't need to be in this PR, if they are the same, shall we extract them into a struct/class, with bit ops implemented in one place?